GoogleSignIn/Tests/Unit/GIDSignInTest.m

```diff
@@ -869,99 +876,89 @@ - (void)testNotHandleWrongPath {
 #pragma mark - Tests - disconnectWithCallback:
 
 // Verifies disconnect calls callback with no errors if access token is present.
-- (void)testDisconnect_accessToken {
+- (void)testDisconnect_accessTokenIsPresent {
   [[[_authorization expect] andReturn:_authState] authState];
   [[[_authState expect] andReturn:_tokenResponse] lastTokenResponse];
   [[[_tokenResponse expect] andReturn:kAccessToken] accessToken];
-  [[[_authorization expect] andReturn:_fetcherService] fetcherService];
+  OCMStub([_authorization fetcherService]).andReturn(_fetcherService);
+  NSData *data = [[NSData alloc] init];
+  [self didFetch:data withToken:kAccessToken error:nil];
+
   XCTestExpectation *accessTokenExpectation =
       [self expectationWithDescription:@"Callback called with nil error"];
   [_signIn disconnectWithCompletion:^(NSError * _Nullable error) {
-    if (error == nil) {
-      [accessTokenExpectation fulfill];
-    }
+    XCTAssertNil(error);
+    [accessTokenExpectation fulfill];
   }];
-  [self verifyAndRevokeToken:kAccessToken
-                 hasCallback:YES
-      waitingForExpectations:@[accessTokenExpectation]];
-  [_authorization verify];
```

Comment:
Why are you removing the calls to `verify`? It looks like you're still using mocks in `GIDSignInTest`, but are removing expectations and verifications that what is expected is called. I think these changes run the risk of making the tests weaker: we are still mocking and and also removing some of the guardrails around the mocks.

Is there a way to no longer mock in these tests (here and throughout)? That is one of the main goals of this refactor.